sdk/openai/openai/test/public/abortsignal.spec.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, describe, beforeEach, it } from "vitest";
+import { matrix } from "@azure-tools/test-utils";
+import { isLiveMode } from "@azure-tools/test-recorder";
+import OpenAI, { AzureOpenAI } from "openai";
+import { createClient } from "./utils/createClient.js";
+import { APIMatrix, APIVersion } from "./utils/utils.js";
+
+describe("AbortSignal", () => {
+  let client: AzureOpenAI | OpenAI;
+
+  matrix([APIMatrix] as const, async function (apiVersion: APIVersion) {
+    beforeEach(async function (context) {
+      // Streaming doesn't work in the record/playback because stream chunks are part of a single response
+      // and the test-proxy just sends all of them at once.
+      if (!isLiveMode()) {
+        context.skip();
+      }
```

Comment:
There is no playback/recording anymore, right?
```suggestion
```

Reply:
I see you want to keep the recording bits, we can leave this then.

Reply:
Yes, I'll add to all test suite to skip the tests in playback mode